agent-nodejs-8/Dockerfile

```diff
@@ -7,18 +7,23 @@ ENV NODEJS_VERSION=8 \
     PATH=$HOME/node_modules/.bin/:$HOME/.npm-global/bin/:$PATH \
     BASH_ENV=/usr/local/bin/scl_enable \
     ENV=/usr/local/bin/scl_enable \
-    PROMPT_COMMAND=". /usr/local/bin/scl_enable"
+    PROMPT_COMMAND=". /usr/local/bin/scl_enable" \
+    CHROME_BIN=/bin/google-chrome
 
 COPY contrib/bin/scl_enable /usr/local/bin/scl_enable
+ADD https://dl.google.com/linux/direct/google-chrome-stable_current_x86_64.rpm google-chrome-stable_current_x86_64.rpm
```

Comment:
yeah, i'm not super concern about this for centos, but we need to sort out whether there's a rhel package even available for this because for the rhel image it absolutely has to be a redhat distributed package.  (basd on @gabemontero's comments below, it sounds like there is not one currently)